DevoxxClientMobile/src/main/java/com/devoxx/DevoxxView.java

```diff
@@ -48,11 +48,11 @@
     public static final AppView EXHIBITOR       = view( ExhibitorPresenter.class,      MaterialDesignIcon.PLACE);
     public static final AppView EXHIBITION_MAPS = view( ExhibitionMapsPresenter.class, MaterialDesignIcon.MAP,                SHOW_IN_DRAWER);
     public static final AppView EXHIBITION_MAP  = view( ExhibitionMapPresenter.class,  MaterialDesignIcon.MAP);
-    public static final AppView SPONSORS        = view( SponsorsPresenter.class,       MaterialDesignIcon.PAYMENT/*,            SHOW_IN_DRAWER*/);
+    public static final AppView SPONSORS        = view( SponsorsPresenter.class,       MaterialDesignIcon.PAYMENT,            SKIP_VIEW_STACK);
```

Comment:
Could you change the icons for SPONSORS and SPONSOR views to match the one that is used for selecting "I'm a sponsor" in the BADGES view?